.github/workflows/nightly.yml

```diff
@@ -14,17 +14,17 @@ jobs:
   build_binary:
```

Comment:
Thanks! Changes to `utils.rs` look good, but can you revert changes to this CI file before we merge?